viewing_party/party.py

```diff
@@ -0,0 +1,123 @@
+def create_movie(title, genre, rating):
+    if title == None or genre == None or rating == None:
+        return None
```

Comment:
The project description says this should happen if any of these values is falsy. A more accurate check would be
```suggestion
    if not title or not genre or not rating:
        return None
```